docs/source/get_started/index.rst

```diff
@@ -5,11 +5,55 @@
 First things first
 ******************
 
-With a fresh AiiDA installation, there are still a few steps to complete, before any calculations can be started:
+With ``aiida-core`` installed, a few steps remain before starting to run calculations:
 
-1) The daemon has to be configured and started
-2) A computer has to be configured
-3) A code has to configured
-4) A plugin has to be installed
+1) :ref:`plugins`
+2) :ref:`setup_computer`
+3) :ref:`setup_code`
+
+.. _plugins:
+
+***************
```

Comment:
why put the `plugins` section in the same file, but use the index to include the ones for setting up a computer and code?

Reply:
I know it's not consistent with the other sections at this point but now at least one sees some relevant information when clicking on "first things first".
Eventually, when code and computer setup have been reasonably shortened, they could all go in one page.

Reply:
fair enough